presto-docs/src/main/sphinx/admin/verifier.rst

```diff
@@ -166,19 +166,70 @@ queries.
     * Checks the nullity of control sum and test sum.
     * If either control mean or test mean very close 0, checks if both are close to 0.
     * Checks the relative error between control sum and test sum.
+* **VARCHAR Columns**
+    * For ``VARCHAR`` columns, a simple checksum column is generated for verification using :func:`!checksum`.
+    * If ``validate-string-as-double`` is set, 6 additional columns below are generated. If ``NULL`` counts
+      are equal before and after casting all values to ``DOUBLE``, apply the aforementioned floating point validation.
+      Otherwise, check if the simple checksum matches.
+        * Count of the ``NULL`` values
```

Comment:
Error in local doc build: 

`/Users/steveburnett/Documents/GitHub/presto/presto-docs/src/main/sphinx/admin/verifier.rst:174: ERROR: Unexpected indentation.`

Suggest unindenting this list, and any nested list within it, and do a local doc build until these similar errors do not appear. See [Building the Documentation](https://github.com/prestodb/presto/tree/master/presto-docs#building-the-documentation) for more information. 

Screenshot of local doc build, showing a vertical gray line in the generated file. 
<img width="799" alt="Screenshot 2024-08-05 at 11 27 58 AM" src="https://github.com/user-attachments/assets/560984ab-b331-40dc-ae02-df7d0474a372">

Reply:
@steveburnett , thanks a lot for pointing to the local build. That's very convenient! I've fixed the build errors.